test/test_ast_generation.py

```diff
@@ -3,6 +3,8 @@
 from pathlib import PurePath
 from typing import Any
 
+import pytest
```

Comment:
I think you need to add `# type: ignore` to this line to make mypy happy. (Unfortunately the pytest API has many items and there are no mypy stubs yet.)